server/app/controllers/applicant/ApplicantRoutes.java

```diff
@@ -200,4 +200,29 @@ public Call confirmAddress(
       return routes.ApplicantProgramBlocksController.confirmAddress(programId, blockId, inReview);
     }
   }
+
+  /**
+   * Returns the route corresponding to the applicant previous block action.
+   *
+   * @param profile - Profile corresponding to the logged-in user (applicant or TI).
+   * @param applicantId - ID of applicant for whom the action should be performed.
+   * @param programId - ID of program to review
+   * @param previousBlockIndex - index of the previous block
+   * @param inReview - true if executing the review action (as opposed to edit)
+   * @return Route for the applicant previous block action
+   */
+  public Call previous(
```

Comment:
**thought:** We prefix our other block-related actions with block like `blockEdit` and `blockReview`. Thoughts on doing that here as well for consistency?

Reply:
Good idea, done!